src/scripts.js

```diff
@@ -126,13 +217,24 @@ function makeRandomDate(userStorage, id, dataSet) {
   return sortedArray[Math.floor(Math.random() * sortedArray.length + 1)].date;
 }
 
+function clearHtml(queryElements) {
+  queryElements.forEach(query => {
+    query.innerHTML = '';
+  })
+}
+
 function addHydrationInfo(id, hydrationInfo, dateString, userStorage, laterDateString) {
-  hydrationToday.insertAdjacentHTML('afterBegin', `<p>You drank</p><p><span class="number">${hydrationInfo.calculateDailyOunces(id, dateString)}</span></p><p>oz water today.</p>`);
```

Comment:
I see this was removed, did we not need it or did it get moved someplace else?

Reply:
Yes. It got moved to addDailyOunces I believe and now addDailyOunces is call there.